pkg/container/bubblewrap_runner.go

```diff
@@ -88,6 +92,11 @@ func (bw *bubblewrap) cmd(ctx context.Context, cfg *Config, debug bool, envOverr
 	if cfg.RunAs != "" {
 		baseargs = append(baseargs, "--unshare-user")
 		baseargs = append(baseargs, "--uid", cfg.RunAs)
+	} else {
```

Comment:
We should do this only if `cfg.RunAs` is not `0` (root)
so if cfg.RunAs != "" and not zero, we use `--uid` and `--gid`, else we completely skip the `--uid` thing

check how I manage it here: https://github.com/chainguard-dev/melange/blob/3d4bb7ed71361e157eab839cb6d650067afdb032/pkg/container/qemu_runner.go#L118

Reply:
So this would standardize the uid to 0 instead of 1000, in both cases rootless, gotcha.
It should remove the inconsistency of the result also, between local and CI that uses sudo with Bubblewrap.

Reply:
So doing a little test, this should be:

```go
baseargs = append(baseargs, "--unshare-user")
if cfg.RunAs != "" {
		baseargs = append(baseargs, "--uid", cfg.RunAs)
		baseargs = append(baseargs, "--gid", cfg.RunAs)
} else {
		baseargs = append(baseargs, "--uid", "0")
		baseargs = append(baseargs, "--gid", "0")
}
```

The inconsistency was that we were skipping the `--unshare-user` when runas is NOT specified
This would fallback to whatever user you are on the host, so in your example 1001

To have consistency we should respect runas when set, and map use the root user in the sandbox if not specified

Reply:
Addressed